donate.html

```diff
@@ -65,9 +65,68 @@
 
 </head>
 <body>
+
 <div class="container-fluid">
   <!-- su'ad -->
-
+  <!-- Header -->
+  <div class = "donate-header">
+    <div style="text-align:center;">
+      <h2>
+        <span style="position: relative">
+        Donating to
+        <div class="d-none d-md-block">
+          <div class="large-spark pink-spark" style="top: -30px; left: -70px;"></div>
+          <div class="med-spark purp-spark" style="top: 30px; left: -120px;"></div>
+          <div class="small-spark blue-spark" style="top: 120px; left: -50px;"></div>
+          <div class="large-spark purp-spark" style="top: -35px; right: -35px;"></div>
+          <div class="small-spark pink-spark" style="top: 150px; right: -25px;"></div>
+          <div class="med-spark blue-spark" style="top: 70px; right: -110px;"></div>
+        </div>
+      </span>
+    </h2>
+      <div> <img src="img/black_colorstripe.png" width="270" height="60"></div>
+      <div class="b1"> <button>Donate</button> </div>
+    </div>
+  </div>
+  <!-- Our Story  -->
```

Comment:
nitpicking: add an empty line to visually separate this section from the header above